src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,43 +1,105 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  handleModalActive: () => void;
+  modalTodo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  handleModalActive,
+  modalTodo,
+}) => {
+  const [user, setUser] = useState<User>({
+    id: 0,
+    name: '',
+    email: '',
+    phone: '',
+  });
```

Comment:
```suggestion
  const [user, setUser] = useState<User | null>(null);
```
better to set the null as the default value